torchci/rockset/pytorch_dev_infra_kpis/__sql/ttrs_percentiles.sql

```diff
@@ -96,11 +114,7 @@ commit_job_durations AS (
     INNER JOIN commons.workflow_run r ON j.run_id = r.id
   WHERE
     1 = 1
-    AND (
-      r.name IN ('pull', 'trunk', 'Lint')
-      OR r.name like 'linux-binary%'
-      OR r.name like 'windows-binary%'
-    )
+    AND r.name = 'pull' -- Stick to pull workflows to reduce noise. Trendlines are the same within other workflows
```

Comment:
I think you could make the workflow name a parameter default to `pull`.  It would be easier to get the numbers for other workflows that way.  IMO, it would be ok to keep track of TTRS for different workflows separately.